### ISSUE TITLE
signin stopped working??

### ISSUE THREAD

**Issue by sirpy** [EddyVerbruggen/cordova-plugin-googleplus#379]
We had google signin working just fine using this package and a couple of days ago it just stoppd working, did any one experience this issue? (web signin still works fine)

**Comment by ChrisEelmaa** [comment]
It is working nicely over here at the very moment. What's the error?

**Comment by remster** [comment]
Same here - working fine.

**Comment by sirpy** [comment]
yeah its working with a new app credentials, must be a specific issue of
our app at google's
